.github/workflows/release.yml

```diff
@@ -132,33 +135,25 @@ jobs:
           export GH_PAGES_DIR="$PWD/gh-pages"
           ./build/dev/publish-plugin-registry-to-gh-pages.sh
 
-          cd ${{ env.GITHUB_WORKSPACE }}
-          message=$(jq -r '.text' mattermost.json)
-          echo "$( jq '.text = "'$message'. publish to GH succeeded."' mattermost.json )" > mattermost.json   
+          echo MM_MESSAGE_APPEND='${{ env.MM_MESSAGE_APPEND }} Published package to npmjs https://www.npmjs.com/package/@eclipse-che/plugin-registry-generator/v/${{ github.event.inputs.version }}.' >> $GITHUB_ENV
       - name: publish to npmjs
         if: ${{ github.event.inputs.publishToNpmJs == 'true' }}
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
         run: |
-          cd ${{ env.GITHUB_WORKSPACE }}
+          cd ${GITHUB_WORKSPACE}
           BUILDER=docker SKIP_FORMAT=true SKIP_LINT=true SKIP_TEST=true ./build.sh --tag "${{ github.event.inputs.version}}"
           cd tools/build
           npm publish --tag latest
 
-          cd ${{ env.GITHUB_WORKSPACE }}
-          message=$(jq -r '.text' mattermost.json)
-          echo "$( jq '.text = "'$message'. publish to npm.js succeeded."' mattermost.json )" > mattermost.json   
+          echo MM_MESSAGE_APPEND='${{ env.MM_MESSAGE_APPEND }} Published package to npmjs https://www.npmjs.com/package/@eclipse-che/plugin-registry-generator/v/${{ github.event.inputs.version }}.' >> $GITHUB_ENV
       - name: Create failure MM message
         if: ${{ failure() }}
         run: |
-          cd ${{ env.GITHUB_WORKSPACE }}
-          message=$(jq -r '.text' mattermost.json)
-          echo "$( jq '.text = ":no_entry_sign: Che Plugin Registry release ${{ github.event.inputs.version }} has failed: https://github.com/eclipse-che/che-plugin-registry/actions/workflows/release.yml. '$message'"' mattermost.json )"  
+          echo "{\"text\":\":no_entry_sign: Che Plugin Registry image and generator ${{ github.event.inputs.version }} release has failed: https://github.com/eclipse-che/che-plugin-registry/actions/workflows/release.yml. ${MM_MESSAGE_APPEND}\"}" > mattermost.json
```

Comment:
should the variable here be global `${{ env.MM_MESSAGE_APPEND }}` instead of local `${MM_MESSAGE_APPEND}` ?